synfig-studio/src/gui/devicetracker.cpp

```diff
@@ -61,23 +61,42 @@ using namespace studio;
 
 /* === M E T H O D S ======================================================= */
 
+static
+GList* get_auxiliary_devices(GdkSeat*& seat) {
+	seat = gdk_display_get_default_seat(
+		gdk_display_manager_get_default_display(
+			gdk_display_manager_get()));
+	return gdk_seat_get_slaves(
+		seat,
+		GDK_SEAT_CAPABILITY_ALL_POINTING );
+}
+			
+static
+GList* get_devices(GdkDevice*& pointer) {
```

Comment:
It would be better to return a tuple/pair with the pointer together with the list. Then the function below without the argument is not necessary, as they do the same thing. The caller would just discard the pointer when it is not interested in using it.